tests/legacy-cli/e2e/assets/protractor-saucelabs.conf.js

```diff
@@ -55,18 +46,9 @@ exports.config = {
       tunnelIdentifier,
     },
     {
-      browserName: 'firefox',
-      version: '48',
-      tunnelIdentifier,
-    },
-    {
-      browserName: 'firefox',
-      version: '60',
-      tunnelIdentifier,
-    },
-    {
-      browserName: 'firefox',
-      version: '68',
```

Comment:
Latest Firefox ESR version is 68: https://www.mozilla.org/en-US/firefox/68.4.2/releasenotes/

Reply:
We don't seem to support FireFox ESR https://angular.io/guide/browser-support#browser-support

Reply:
I’d consider it technically a latest version of Firefox; making Firefox essentially have two. The ESR latest version is used extensively in corporate environments (intranet applications, etc.) due to its support and stability guarantees.
There have been reported issues in the past concerning support of previous ESR versions.

@IgorMinar thoughts?

Reply:
I'm fine adding it, but then we should also document it and add it to the test suite no the framework repo.